src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dto/IncorporationSubResourcesDto.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class IncorporationSubResourcesDto {
+
+    @JsonProperty("general_partners")
+    private List<GeneralPartnerDto> generalPartners;
+
+    @JsonProperty("limited_partners")
+    private List<LimitedPartnerDto> limitedPartners;
+
+    @JsonProperty("partnership")
+    private LimitedPartnershipSubmissionDto partnership;
```

Comment:
should we change the name of LimitedPartnershipSubmissionDto to PartnershipDto ? it would be in line with the other dtos then

Reply:
better in a separate task

Reply:
Created this new story to cover this - https://companieshouse.atlassian.net/browse/LP-468.